src/main/CommandHandler.cpp

```diff
@@ -1187,11 +1249,22 @@ CommandHandler::generateLoad(std::string const& params, std::string& retStr)
 
         uint32_t numItems = isCreate ? cfg.nAccounts : cfg.nTxs;
         std::string itemType = isCreate ? "accounts" : "txs";
-
-        retStr +=
+        Json::Value res;
+        res["status"] =
             fmt::format(FMT_STRING(" Generating load: {:d} {:s}, {:d} tx/s"),
```

Comment:
I think it would be good to make this status more informative given all the new modes (this can be done later when the API settles)